### PR TITLE
Added jax.numpy.interp

### LINKED ISSUE #13870: interp

### PR DESCRIPTION
**Pull request by abhijeetsharma200**
Close #13870

### CODE REVIEW

**Comment by abhijeetsharma200** [comment]
@karalleyna I would appreciate any guidance you can give me on this.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
